### PR TITLE
Overhauled entirely

### PR DESCRIPTION
**Pull request by VickyRampin**
I overhauled the NYU subdomain for consistency with the main reproduciblescience.org site, and to update a lot of information that was out of date. 

View live: https://vickysteeves.github.io/reproducible-science-nyu

### CODE REVIEW

File: bibtex/working-group.bib [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+	year = {2018}
+}
+
+@misc{rampin_repromatch_nodate,
+	title = {{ReproMatch}},
+	url = {http://repromatch.poly.edu/tools/search/},
+	abstract = {Many tools have been developed to support researchers in the practice of reproducible research. However, given the many requirements across disciplines, identifying the right tool or a set of tools can be a daunting task. With the help of developers of reproducibility tools, we will create a comprehensive catalog of tools. Using ReproMatch, users will be able to search for tools that match their reproducibility needs.},
+	year={2018},
+	journal = {ReproMatch},
+	author = {Rampin, Rémi and Ellqvist, Tommy}
```
Comment (remram44) [inline bibtex/working-group.bib]: I didn't work on ReproMatch, you can see the team on http://repromatch.poly.edu/about/

It's at least 3 years old, @fchirigati?

File: bibtex/working-group.bib [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+	title = {{ReproZip}},
+	copyright = {BSD-3-Clause},
+	shorttitle = {reprozip},
+	url = {https://github.com/ViDA-NYU/reprozip},
+	abstract = {ReproZip is a tool that simplifies the process of creating reproducible experiments from command-line executions, a frequently-used common denominator in computational science},
+	publisher = {ViDA-NYU},
+	author = {Rampin, Rémi and Chirigati, Fernando and Freire, Juliana},
+	collaborator = {Steeves, Vicky},
+	month = {June},
+	year = {2018}
```
Comment (remram44) [inline bibtex/working-group.bib]: ReproZip 1.0 was September 30, 2015

**Comment by remram44** [comment]
It looks like a lot of the bib entries defaulted to 2018 (some show up as "June 2018")

File: bibtex/working-group.bib (hunk 1 above)
Comment (VickyRampin) [reply bibtex/working-group.bib]: Ah, sorry, I went on the GitHub contributors (it's just you and Tommy, @remram44).

Comment (fchirigati) [reply bibtex/working-group.bib]: Yep, we released the website on March 2013.

Comment (VickyRampin) [reply bibtex/working-group.bib]: Ok, will fix!

**Comment by VickyRampin** [comment]
This last commit should fix those issues with dates in the bib file.

File: bibtex/working-group.bib [hunk 3]
```diff
@@ -0,0 +1,601 @@
...
+}
+
+@unpublished{cranmer_cern_2017,
+	address = {CERN},
+	title = {{CERN} {Analysis} {Preservation}/{DASPOS}/{RECAST} workshop},
+	url = {https://indico.cern.ch/event/611389/},
+	abstract = {The aims of this mini workshop are: 1) getting community members together to give feedback on the first prototype of CERN Analysis Preservation (CAP) and its connection with RECAST and DASPOS work, 2) special emphasis will be given to the three high-level use cases of CAP: describe, capture and reuse a physics analysis. Dedicated sessions will allow a more detailed discussion of functionalities and expectations from the community to inform the roadmap of the services/tools/integrations. Based on the demonstration of the recent prototype most of the day will be spent on a discussion of the next steps. Examples include, but are not limited to: Analysis submission, are we missing out on any types of analysis, What is needed (beyond RECAST) for reuse/reinterpretation, what is needed or interesting to do on top of CAP, how can tools be connected (for example connected to CAP), search terms of interest, etc.},
+	author = {Cranmer, Kyle and Heinrich, Lukas},
+	note = {DASPOS \@ CERN},
```
Comment (remram44) [inline bibtex/working-group.bib]: This backslash shows up

File: bibtex/working-group.bib [hunk 4]
```diff
@@ -0,0 +1,601 @@
...
+
+@misc{group_reproducible_2018,
+	title = {Reproducible {Science}},
+	url = {https://reproduciblescience.org/},
+	abstract = {On this site, the moderators have curated sources of various types discussing reproducibility in the directory. You can find academic papers, blog posts, popular media articles, talks, tools, and more.},
+	language = {en},
+	journal = {Reproducible Science},
+	author = {Group, NYU Reproducibility Working},
+	month = {June},
+	year = {2018}
```
Comment (remram44) [inline bibtex/working-group.bib]: Should this also be January 2017? Or August 2015, since this is when you set up the wordpress version.

**Comment by VickyRampin** [comment]
Fixed!

**Comment by remram44** [comment]
Only rs.org still says June 2018

**Comment by remram44** [comment]
Also there is no link between [reproduciblescience.org](https://reproduciblescience.org/) and [nyu.reproduciblescience.org](https://nyu.reproduciblescience.org/)?

**Comment by VickyRampin** [comment]
Ok, I actually fixed it now. No, there's no link between the two,

**Pull request merged**